fl4health/parameter_exchange/fedpm_exchanger.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+import torch
+import torch.nn as nn
+from flwr.common.typing import Config, NDArrays
+
+from fl4health.parameter_exchange.layer_exchanger import DynamicLayerExchanger
+from fl4health.utils.functions import sigmoid_inverse
+
+
+class FedPmExchanger(DynamicLayerExchanger):
+    def pull_parameters(self, parameters: NDArrays, model: nn.Module, config: Optional[Config] = None) -> None:
+        current_state = model.state_dict()
+        layer_params, layer_names = self.unpack_parameters(parameters)
+        for layer_name, layer_param in zip(layer_names, layer_params):
+            # Apply the inverse of the Sigmoid function
+            # since the scores for masked layers are supposed to be unbounded.
+            with torch.no_grad():
+                current_state[layer_name] = sigmoid_inverse(torch.tensor(layer_param))
```

Comment:
I also want to point out that it is actually important to apply this function to ensure that the parameter scores are unbounded (i.e., not bounded between 0 and 1), since during training there is no guarantee that the scores will remain between 0 and 1 after gradient updates.